yellowbrick/features/pca.py

```diff
@@ -215,7 +216,7 @@ def finalize(self, **kwargs):
 
 def pca_decomposition(X, y=None, ax=None, features=None, scale=True,
                       proj_dim=2, proj_features=False, color=None,
-                      colormap=palettes.DEFAULT_SEQUENCE,
+                      colormap=palettes.DEFAULT_SEQUENCE,alpha=1,
```

Comment:
place a space after the comma right before alpha

Reply:
OK. Will do so!